envcfg_test.go

```diff
@@ -67,6 +67,22 @@ type cfgValid2 struct {
 	INT       validType
 }
 
+type validTextUnmarshaler struct {
+	DummyFloat float64
+}
+
+func (s *validTextUnmarshaler) UnmarshalText(text []byte) error {
+	s.DummyFloat = 1.0
```

Comment:
@nsaje here you only test if this method is called. Would you mind changing what this method does (and the corresponding tests you wrote), so we'll also test that the correct value get's passed to `UnmarshalText`?